Fix: UI issues in announcement and user experience tables (Closes #16, #17)

Triveni-Balekundri opened the pull request:
🔖 PR Title:
Fix: UI issues in announcement and user experience tables (Closes #16, #17)

📄 Description:
This pull request fixes the UI-related  improvement and the user experience for the announcement and user experience tables. It also includes backend preparation using PHP for future database integration.

✅ Documentation 📝
✅ UI/UX Update 🎨
✅ enhancement

https://github.com/user-attachments/assets/694295a2-3f04-41f5-9f37-c50f1282d13d



🧑‍💻 What changes were made?
1. Removed Google Script-based embedding and replaced with standard HTML src files named: anouncement-table.htm and user-experience-table.html in iframes.

2. Ensured table UI works properly across devices (responsive fix).

3. Added commented PHP code for future database access (currently inactive, serves as a reference or placeholder for backend connection).


🖼️ Screenshot or Screen Recording (max 15 seconds)
📸 Attached below:


💡 Tech Stack Used:
HTML 5 / CSS
PHP (commented code for backend logic)

🔗 Issue Reference:
Closes #16, #17

🧩 Contribution Type:
 Frontend 🎯
 Backend 🔙 (PHP placeholder)
 UI/UX 🎨

🙏 Additional Info:
PHP code is not currently active but provided for future dynamic data integration.
No breaking changes introduced.


video:
https://github.com/user-attachments/assets/905fb7e8-bb23-429a-ac71-f5e69e0bb888

image:
<img width="1923" height="877" alt="Screenshot 2025-07-23 135402" src="https://github.com/user-attachments/assets/98f7841e-d813-49f3-b12d-e69b95665432" />

VAIBHAVBABELE commented:
> Hi @VAIBHAVBABELE, I’ve created a PR for fixing the announcement table and user experience table issues (#16 and #17 similar). Kindly review and provide feedback. I’ve also added commented PHP code for future DB integration. Thank you!

Are you add the php and mysql query and thier data for dynamic form. Store the required all code related this work.

Triveni-Balekundri commented:
> > Hi @VAIBHAVBABELE, I’ve created a PR for fixing the announcement table and user experience table issues (#16 and #17 similar). Kindly review and provide feedback. I’ve also added commented PHP code for future DB integration. Thank you!
> 
> Are you add the php and mysql query and thier data for dynamic form. Store the required all code related this work.

Replying to this:
Yes, I’ve added the PHP and MySQL code as comments in the HTML file to show how the dynamic form would work. Since I don’t have access to your college’s database, I couldn’t test or implement the backend functionality fully. My main task was to fix the announcement table UI, which is complete and further i will make that one change of alignment and then push this pr.
Thank you !

Triveni-Balekundri commented:
> @Triveni-Balekundri all things are good, make small update heading align centre. Then push this pr.

The heading is already aligned to the center. Please let me know if you're seeing it differently or if there's a specific styling issue on your end. Thank you.

Triveni-Balekundri commented:
pls add labels for this issue.
